staging/prometheus-operator/templates/prometheus/psp.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: {{ template "prometheus-operator.fullname" . }}-prometheus
-  namespace: {{ $.Release.Namespace }}
```

Comment:
we do not need namespace here?

Reply:
same as upstream: https://github.com/helm/charts/blob/master/stable/prometheus-operator/templates/prometheus/psp.yaml#L5-L7

that said, here is the history for why they removed it https://github.com/helm/charts/commit/30b43ea0feea26755677bb07b8b6df9c4b0193f9